cloudfront/invalidation/lambda/.prettierrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  trailingComma: 'es5',
+  singleQuote: true,
+};
```

Comment:
We should set this universally at the root.

Reply:
Similar to `.gitignore` reply - should we move `.prettierrc.js` for all?

Reply:
I think that it's not blocking for this PR, though a good idea for a Wellcome dev to do this in a future PR.